### ISSUE TITLE
Bronchial hemorrhage - PULMO

### ISSUE THREAD

**Issue by katrinknoflach** [obophenotype/human-phenotype-ontology#7078]
**Preferred term label:**
Bronchial hemorrhage

**Synonyms**
Bronchial bleeding

**Definition (free text, please give PubMed ID)**
Bronchial hemorrhage is a focal bleeding located in the bronchial system. It can be diagnosed by tracheobronchoscopy.

**Parent term (use hpo.jax.org/app)**
Abnormal bronchus morphology

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

@crapp, @mgriese

**Comment by pnrobinson** [comment]
added
